packages/base/src/commands.ts

```diff
@@ -878,6 +878,56 @@ export function addCommands(
     icon: targetWithCenterIcon
   });
 
+  commands.addCommand(CommandIDs.newDrawVectorLayer, {
+    label: trans.__('Create New Draw Vector Layer'),
+    isToggled: () => {
+      const selectedLayer = getSingleSelectedLayer(tracker);
+      if (!selectedLayer) {
```

Comment:
In case there is no selected layer, what would make the most sense in terms of UX would be to automatically create a new GeoJSONSource (with an associated layer) and start editing that layer.